common/scala/src/main/scala/whisk/core/entity/Exec.scala

```diff
@@ -278,21 +278,33 @@ protected[core] object Exec extends ArgNormalizer[Exec] with DefaultJsonProtocol
 
           manifest.attached
             .map { a =>
-              val jar: Attachment[String] = {
-                // java actions once stored the attachment in "jar" instead of "code"
-                obj.fields.get("code").orElse(obj.fields.get("jar"))
+              // java actions once stored the attachment in "jar" instead of "code"
+              val code = obj.fields.get("code").orElse(obj.fields.get("jar"))
```

Comment:
perhaps as part of the schema migration pr, you can normalize all `jar` occurrences to `code` and we can remove this legacy bit.

Reply:
As part of https://github.com/apache/incubator-openwhisk/pull/2938 I have normalized jar to code. Will have to remove the jar reference here.